src/main/java/com/gregtechceu/gtceu/data/lang/MachineLang.java

```diff
@@ -515,6 +515,7 @@ protected static void init(RegistrateLangProvider provider) {
         provider.add("gtceu.multiblock.cleanroom.clean_state", "Status: §aCLEAN");
         provider.add("gtceu.multiblock.cleanroom.dirty_state", "Status: §4CONTAMINATED");
         provider.add("gtceu.multiblock.cleanroom.clean_amount", "Cleanliness: §a%s%%");
+        provider.add("gtceu.multiblock.dimensions", "Dimensions: %s§cw§r, %s§ah§r, %s§bd§r");
```

Comment:
IMO the dimensions should be on separate lines and indented a bit. If you don't want that, just make them a bit clearer:
```suggestion
        provider.add("gtceu.multiblock.dimensions", "Dimensions: width: %s, height: %s, depth: %s");
```
Or
```suggestion
        provider.add("gtceu.multiblock.dimensions", "Dimensions: §c§l%s§rx§a§l%s§rx§9§l%s§r");
```